modules/ossm-migrating-a-multitenant-deployment.adoc

```diff
@@ -0,0 +1,179 @@
+
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-migration-guides-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrating-a-multitenant-deployment_{context}""]
+== Migrating a multitenant deployment
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.14 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
+* You have completed the premigration checklists.
+* You have the {SMProduct} {SMv2Version} Operator installed.
+* You have the {SMProduct} 3 Operator installed.
+* You have `IstioCNI` installed.
+* You have `istioctl` installed.
+* You are running a `MultiTenant` `ServiceMeshControlPlane`.
+
+.Procedure
+
+The `bookinfo` example application is being used for demonstration purposes. You can follow these same steps with your own workloads.
+
+. Create your `Istio` resource.
++
+.Example `Istio` resource
++
+[source,yaml]
+----
+   apiVersion: sailoperator.io/v1alpha1
+   kind: Istio
+   metadata:
+     name: istio-tenant-a
+   spec:
+     namespace: istio-system-tenant-a <1>
+     version: v1.23.0
```

Comment:
Is it possible to use some variable? Istio version for GA will be probably 1.24.3